api/taxonomies/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers as ser
+
+from api.base.serializers import JSONAPISerializer
+
+
+class TaxonomySerializer(JSONAPISerializer):
+    filterable_fields = frozenset([
+        'text',
+        'parent_ids',
+        'type',
+        'id'
+    ])
+
+    type = ser.CharField(max_length=200)
+    text = ser.CharField(max_length=200)
+    parent_ids = ser.ListField(child=ser.CharField(max_length=50))
```

Comment:
I haven't gotten that far yet, but based on the way taxonomies are usually formed, I'm assuming that A) a `Taxonomy` object is intended to represent a leaf in a tree-structured representation of possible taxonomic classifications for Preprints or files therein and that B) this field will be pointing to `_id` fields of other `Taxonomy` objects? 

If so, either this field is about 150 characters to short or the definition on the line below is 150 characters too long.